ax/utils/sensitivity/sobol_measures.py

```diff
@@ -423,8 +423,14 @@ def __init__(
 
         def input_function(x: Tensor) -> Tensor:
             with torch.no_grad():
-                p = checked_cast(GPyTorchPosterior, self.model.posterior(x))
-            return link_function(p.mean, p.variance)
+                means, variances = [], []
+                # Since we're only looking at mean & variance, we can freely
+                # use mini-batches.
+                for x_split in x.split(split_size=32):
+                    p = checked_cast(GPyTorchPosterior, self.model.posterior(x_split))
+                    means.append(p.mean)
+                    variances.append(p.variance)
```

Comment:
Can we make this split size configurable as an input? 32 may be rather small in cases where memory isn't really an issue. Ideally we can even use some heuristic to determine this batching factor based on the type of model being used here (though that's a separate question beyond this PR).

Reply:
Yeah, this is definitely on the conservative side. Bernie's change was using a much larger value and still not using excess amounts of memory. I can make it configurable and bump up the default a bit. This is just an `n x d` tensor evaluated with `no_grad`, so the memory usage will generally be quite low.